src/AcmDropdown/AcmDropdown.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('AcmDropdown', () => {
     const onSelect = jest.fn()
     const onHover = jest.fn()
     const Component = (props: ComponentProps) => {
-        const dropdownItems = [
+        const dropdownItems: AcmDropdownItems[] = [
```

Comment:
When you added the type here, did you get a warning for `label` and `labelColor` properties? They are not listed in AcmDropdownItems. 

Also, can you update `isDisabled` here to use `isAriaDisabled`?

Thanks

Reply:
I did get warnings, so I added the properties to AcmDropdownItems in the AcmDropdown.tsx file. Should be listed as one of the changes I made.